source/reference/configuration.txt

```diff
@@ -0,0 +1,110 @@
+.. _c2c-config:
+
+Configuration File Settings
+###########################
+
+.. include:: /includes/preview-warning
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can configure :program:`mongosync` instances at startup using a configuration
+file.  The configuration file contains settings that are the equivalent of
+``mongosync`` command-line options.  See :ref:`c2c-config-setting-option-map`.
+
+
+Configuration File
+==================
+
+Most :program:`mongosync` command-line options can be written to file.
+The configuration file specifies values for each setting using YaML format.
+For example:
+
+.. code-block:: yaml
```

Comment:
now that we have expanded the config file example with it's own page, do we still need this code-block example here? In this section, should we also just link to that page for more information?

Reply:
@alan-zheng-mdb, Could you clarify on this point?  I don't see a page with an expanded config file example in the build.

Reply:
Fixed